### DISCUSSION TITLE
Flutter Mobile SDK - Contribution

### DISCUSSION THREAD

**Discussion by sudalaimuthu-a** [grafana/faro-web-sdk#538] [General]
We are in the process of creating a Flutter Mobile SDK capable of gathering various application metrics, such as:

1. Tracking app cold and warm starts.
2. Monitoring CPU and memory usage.
3. Capturing refresh rates.
4. Logging asset information.
5. Recording network resource usage.
6. Reporting errors and crashes.
7. Logging custom events.
8. Gathering device information.

Our intention is to open-source this SDK. Could you please provide guidance on the process for contributing to open-source projects?

![image](https://github.com/grafana/faro-web-sdk/assets/124167819/2f281c87-0ebb-4ecf-9d30-ff7013d2b3ad)

CC: @NasarTarique, @ska663

**Comment by codecapitano** [comment]
Hey @Saravanan, amazing work!

Our PM @cedricziel  and myself recently talked briefly about how to provide/coordinate/integrate/... community contributions.
So my proposal would be, let's wait till next week so we can come up with a good idea on how to best handle this.

Again, great work ❤️
